src/ucp/wireup/address.c

```diff
@@ -312,16 +319,66 @@ static void ucp_address_memcheck(ucp_context_h context, void *ptr, size_t size,
     }
 }
 
+static unsigned ucp_address_pack_flags(uint64_t packed_flags,
+                                       uint64_t cap_flags,
+                                       unsigned start_bit,
```

Comment:
- can be `uint8_t`
 - `assert(start_bit < 64);`

Reply:
done